doc/api/net.md

```diff
@@ -1125,6 +1129,8 @@ This property represents the state of the connection as a string.
 * If the stream is readable and writable, it is `open`.
```

Comment:
It might be nice to change `stream` here to `socket` as it makes more sense IMO.